components/service/glean/src/main/java/mozilla/components/service/glean/config/Configuration.kt

```diff
@@ -0,0 +1,18 @@
+package mozilla.components.service.glean.config
```

Comment:
nit: license header here and in the other files :)

Reply:
What is wrong with my memory!!? 🤦‍♂️ Addressing this :D